schutzbot/save_journal.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# use tee, otherwise shellcheck complains
+sudo journalctl --boot | tee journal-log >/dev/null
+
+# As it might contain sensitive information and is important for debugging
+# purposes, encrypt journal-log with a symmetric passphrase.
+gpg --batch --yes --passphrase "$GPG_SYMETRIC_PASSPHRASE" -o journal-log.gpg --symmetric journal-log
```

Comment:
```suggestion
gpg --batch --yes --passphrase "$GPG_SYMMETRIC_PASSPHRASE" -o journal-log.gpg --symmetric journal-log
```
A bit of a nitpick but if there's a typo in the variable name I'm sure it'll trip someone up in the future

Reply:
Indeed, good catch.